vpr/src/pack/cluster.cpp

```diff
@@ -572,6 +574,10 @@ std::map<t_logical_block_type_ptr, size_t> do_clustering(const t_packer_opts& pa
 
             VTR_LOGV(verbosity > 2, "Complex block %d:\n", num_clb);
 
+            /*this temp structure is used to store updates to the cluster's
```

Comment:
update comment, explain used to store updates to cluster's partitionregion as we try to add primitives to it. since some of the primitives might fail legality, we need this temp structure to store the legality